internal/obsreportconfig/obsreportconfig.go

```diff
@@ -88,6 +88,14 @@ func allViews() *ObsMetrics {
 	tagKeys = []tag.Key{obsmetrics.TagKeyExporter}
 	views = append(views, genViews(measures, tagKeys, view.Sum())...)
 
+	errorNumberView := &view.View{
+		Name:        "exporter/send_failed_requests",
```

Comment:
Use the prefix constant to ensure no typos

Reply:
do you mean use `ExporterPrefix` like other views? actually we don't really care which exporter does the failure come from, so i think we can keep use this general view name.